app/grants/models/clr_match.py

```diff
@@ -8,14 +8,26 @@ class CLRMatch(SuperModel):
     """Define the structure of a CLR Match amount."""
 
     round_number = models.PositiveIntegerField(blank=True, null=True)
-    amount = models.FloatField()
     grant = models.ForeignKey(
         'grants.Grant',
         related_name='clr_matches',
         on_delete=models.CASCADE,
         null=False,
         help_text=_('The associated Grant.'),
     )
+    token = models.ForeignKey(
+        'economy.Token',
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+    )
+    amount = models.FloatField(
+        help_text='Token Amount'
+    )
+    usd_amount = models.PositiveIntegerField(
+        null=True,
+        blank=True
```

Comment:
worth adding help_text to say this is the USD amount ?

Reply:
renaming this would break things as earlier we would rely on amount as the usd_amount.
We might have to explore places and update usd_amount wherever applicable